client/views/msg_preview.tpl

```diff
@@ -8,11 +8,15 @@
 	{{else}}
 		{{#if thumb}}
 			<img src="{{thumb}}" class="thumb">
+			<div class="toggle-text">
```

Comment:
Probably worth rendering this div whether or not the thumb appears for consistency sake.